app/views/layouts/application.html.erb

```diff
@@ -1,27 +1,29 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>KhanBurmese</title>
-  <%= favicon_link_tag '/favicon.ico' %>
+  <title>KhanAcademy Myanmar</title>
+  <%= favicon_link_tag '/favicon.png' %>
   <%= stylesheet_link_tag    "application", :media => "all" %>
   <%= javascript_include_tag "application" %>
   <%= csrf_meta_tags %>
+  <meta name="viewport" content="width=500, initial-scale=1">
 </head>
 <body>
 
 <!-- Navigation bar. -->
 <nav class="navbar navbar-inverse navbar-fixed-top" role="navigation">
   <div class="container">
-    <div>
-      <%= image_tag '/favicon.ico', :class => 'img-responsive img-logo' %>
-      <%= link_to 'KhanAcademy Myanmar', root_path, :class => 'navbar-brand pull-left inline-brand' %>
-    </div>
-    <div class="collapse navbar-collapse">
+    <%= link_to(root_path) do %>
+      <%= image_tag('/favicon.png', :class => 'img-responsive img-logo') %>
+      <div class="navbar-brand"> <b>Khan</b>Academy <b><i>Myanmar</i></b></div>
+    <% end %>
+    <div class="pull-right">
       <% if user_signed_in? %>
         <ul class="nav navbar-nav navbar-right">
           <li class="dropdown">
             <a href="#" class="dropdown-toggle" data-toggle="dropdown"><%= current_user.email %> <span class="caret"></span></a>
             <ul class="dropdown-menu" role="menu">
+              <li><a href="#">Settings</a></li>
```

Comment:
If we decide to add additional dropdown links such as "profile," consider using a divider to group these links. Refer to the following:
`<li role="presentation" class="divider"></li>`